style.css

```diff
@@ -845,3 +844,43 @@ textarea::placeholder {
     margin-left: 24px;
   }
 }
+
+.mobile-menu-list {
+  display: none;
+  flex-direction: column;
+  align-items: flex-start;
+  list-style: none;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 57px;
+  background: #3c3a39;
+  mix-blend-mode: normal;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  height: 100vh;
+}
+
+.margin-li {
+  margin-top: 119px;
+}
+
+.mobile-menu-list-li {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  z-index: 1;
+  color: #fff5e1;
+  margin-bottom: 40px;
+  border-bottom: 2px solid #6f6c6b;
```

Comment:
- [OPTIONAL] The border bottom covers almost the entire length of the screen, please kindly fix that. See image below.

***The design***
![atilaD](https://user-images.githubusercontent.com/65852621/136783933-0597fea9-fe02-484a-838f-5c16cecbadfe.png)